challenge_3/python/mindm/src/majority.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import sys
+from collections import defaultdict
+
+
+def args_to_list(arg_string):
+    """ Parses argument-string to a list
+    """
+    # Strip whitespace -> strip brackets -> split to substrings ->
+    # -> strip whitespace
+    arg_list = [x.strip() for x in arg_string.strip().strip("[]").split(',')]
+    return arg_list
+
+
+def test_digit(arr):
+    """ Exits if list contains non-numeric strings
+    """
+    for element in arr:
+        if not element.isdigit():
+            print("Error: '{}' is not numeric.".format(element))
+            exit(1)
+
+
+if len(sys.argv) == 1:              # Arguments provided by piping in shell

```

Comment:
Its good practice to handle argument vectors in the beginning of the program instead of running half of your program then handle arguments.